### PR TITLE
Fix adding user properties to UserInfo context

### PR DESCRIPTION
**Pull request by Arciiix**
I have noticed another bug related to user preferences. #242 that I pushed yesterday fixes saving those user preferences to the database, but they are still not loaded into the `UserInfo` context, therefore they are not accessible from any of the messaging integrations.

To reproduce, you can again set up the Telegram integration and add a breakpoint here:
https://github.com/notifo-io/notifo/blob/fe94068b9a1eddcdecc42cbef4111656b605d921/backend/src/Notifo.Domain.Integrations/Telegram/TelegramIntegration.Messaging.cs#L25-L33

No matter what the user preferences are, the `chatId` will always be empty (because it doesn't exist in the `Properties` of `UserInfo` instance).

My fix merges the `SystemProperties` field together with the `UserProperties`, prioritizing the `SystemProperties` (therefore the ```.Last()```) - because they were what was previously being set there - to ensure this fix doesn't break anything.

All the tests related to this are passing.

PS: I'm finishing the Discord integration. I'm doing those small pull requests that are general fixes not related to the integration itself, so that those changes are atomic (as you suggested in #38 ).

Have a nice day!

### CODE REVIEW

File: backend/src/Notifo.Domain/Integrations/IntegrationExtensions.cs [hunk 1]
```diff
@@ -6,13 +6,25 @@
...
 {
     public static UserInfo ToContext(this User user)
     {
-        return new UserInfo { Id = user.Id, EmailAddress = user.EmailAddress, PhoneNumber = user.PhoneNumber, Properties = user.SystemProperties };
+        return new UserInfo
+        {
+            Id = user.Id,
+            EmailAddress = user.EmailAddress,
+            PhoneNumber = user.PhoneNumber,
+            Properties = user.Properties
```
Comment (SebastianStehle) [inline backend/src/Notifo.Domain/Integrations/IntegrationExtensions.cs:22]: I am not a big fan of LINQ in this case. Can we just make it with a normal dictionary like

var properties = new Dictionary<string, string>(user.Properties);

if (user.SystemProperties != null)
{
   foreach (var (key, value) in user.SystemProperties)
   {
      properties[key] = values;
   } 
}

**Review by SebastianStehle — COMMENTED** [review]
See my comment

**Comment by SebastianStehle** [comment]
Btw: Notifo test coverage is really bad. Feel free to add new tests

File: backend/src/Notifo.Domain/Integrations/IntegrationExtensions.cs (hunk 1 above)
Comment (Arciiix) [reply backend/src/Notifo.Domain/Integrations/IntegrationExtensions.cs:22]: Sure, of course. Pushing the new changes...

**Pull request merged**